engine/physics/src/physics/physics_3d.cpp

```diff
@@ -635,7 +635,7 @@ namespace dmPhysics
         switch(shape->getShapeType())
         {
             case SPHERE_SHAPE_PROXYTYPE:        return new btSphereShape(((btSphereShape*)shape)->getRadius()); break;
-            case BOX_SHAPE_PROXYTYPE:           return new btBoxShape(((btBoxShape*)shape)->getHalfExtentsWithoutMargin()); break;
+            case BOX_SHAPE_PROXYTYPE:           return new btBoxShape(((btBoxShape*)shape)->getHalfExtentsWithMargin()); break;
```

Comment:
Since the margin is removed within the constructor, we need the size including the margin.